fastlane/AppFile

```diff
@@ -0,0 +1,4 @@
+app_identifier "io.cozy.files"
```

Comment:
Because it's app to display files ;) but we choose this really speed with @poupotte 

If you prefer change it it's now because after publish it's not possible to modify it.

- io.cozy.mobile.files
- io.cozy.mobile.sync

?